sql/memo/memo.go

```diff
@@ -38,12 +38,15 @@ type TableAndColumn struct {
 // an exprGroup, produce the same rows (possibly unordered) and schema.
 // Physical plans are stored in a linked list within an expression group.
 type Memo struct {
-	cnt         uint16
-	root        *ExprGroup
+	cnt  uint16
+	root *ExprGroup
+
+	// todo: deprecated
```

Comment:
What's stopping these fields from being deleted now? Make a note

Reply:
it might not look like it, but I was trying to make this PR have less changes

Reply:
I can delete it all now but was going to do in follow up